[IMP] stock_currency_valuation: Make standard_price_in_currency readonly

fw-bot-adhoc opened the pull request:
Forward-Port-Of: ingadhoc/stock#795

feg-adhoc committed:
[IMP] stock_currency_valuation: Make standard_price_in_currency readonly

X-original-commit: 850ad67e57d4cf2c1c05844c7bb732cdbb004308

roboadhoc commented:
[![Pull request status dashboard](https://mergebot.adhoc.ar/ingadhoc/stock/pull/800.png)](https://mergebot.adhoc.ar/ingadhoc/stock/pull/800)

fw-bot-adhoc commented:
@feg-adhoc @rov-adhoc cherrypicking of pull request ingadhoc/stock#795 failed.

stdout:
```
CONFLICT (modify/delete): stock_currency_valuation/__manifest__.py deleted in 0d3e5e7eea336d4475e33ea9cedecc90499ec194 and modified in 4361b502bd8e7b865749fcad3f65f3a969d880b3.  Version 4361b502bd8e7b865749fcad3f65f3a969d880b3 of stock_currency_valuation/__manifest__.py left in tree.
CONFLICT (modify/delete): stock_currency_valuation/models/product_product.py deleted in 0d3e5e7eea336d4475e33ea9cedecc90499ec194 and modified in 4361b502bd8e7b865749fcad3f65f3a969d880b3.  Version 4361b502bd8e7b865749fcad3f65f3a969d880b3 of stock_currency_valuation/models/product_product.py left in tree.
CONFLICT (modify/delete): stock_currency_valuation/models/product_template.py deleted in 0d3e5e7eea336d4475e33ea9cedecc90499ec194 and modified in 4361b502bd8e7b865749fcad3f65f3a969d880b3.  Version 4361b502bd8e7b865749fcad3f65f3a969d880b3 of stock_currency_valuation/models/product_template.py left in tree.
CONFLICT (modify/delete): stock_currency_valuation/views/product.xml deleted in 0d3e5e7eea336d4475e33ea9cedecc90499ec194 and modified in 4361b502bd8e7b865749fcad3f65f3a969d880b3.  Version 4361b502bd8e7b865749fcad3f65f3a969d880b3 of stock_currency_valuation/views/product.xml left in tree.

```

Either perform the forward-port manually (and push to this branch, proceeding as usual) or close this PR (maybe?).

In the former case, you may want to edit this PR message as well.

:warning: after resolving this conflict, you will need to merge it via @roboadhoc.

More info at https://github.com/odoo/odoo/wiki/Mergebot#forward-port
